src/FreeAge/common/player.hpp

```diff
@@ -135,3 +136,36 @@ struct PlayerStats {
   /// If a building has ever been constructed per building type.
   bool buildingExisted[static_cast<int>(BuildingType::NumBuildings)];
 };
+
+struct Player {
+
+  /// TODO: pass the civilization
+  /// TODO: rename int playerColorIndex to colorIndex
+  Player(int index, int playerColorIndex, const GameData& gameData);
+
+  inline const PlayerStats& GetPlayerStats() const { return stats; };
+  inline PlayerStats& GetPlayerStats() { return stats; };
+
+  inline const UnitTypeStats& GetUnitStats(UnitType unitType) const { return unitTypeStats.at(static_cast<int>(unitType)); }
+  inline const BuildingTypeStats& GetBuildingStats(BuildingType buildingType) const { return buildingTypeStats.at(static_cast<int>(buildingType)); }
+
+  /// The player index.
```

Comment:
Comment on what this index is for (which array(s) is this index used for, maybe mention `match->GetPlayerIndex`)?